src/style.css

```diff
@@ -0,0 +1,104 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.searchbar {
+  position: absolute;
```

Comment:
that is the cause preventing your app from passing tests
do not use absolute positioning, it literally makes your elements stuck in one place, as a result – it is not possible to dress them up with paddings and margins
what you can do – for example, you can try applying horizontal paddings to your forms so that they do not adhere to the left/right sides